BUILDING.md

```diff
@@ -106,7 +106,9 @@ You need to install the following
 - Xcode
 - [Doxygen](#doxygen) (only if generating documentation)
 
-For the load tests applications you need to install the [Vulkan SDK](#vulkan-sdk).
+For the load tests applications you need to install the Vulkan SDK and run the included setup-env.sh or
+otherwise create an environment variable `VULKAN_SDK` pointing to the macOS directory within the directory where you installed the SDK.
+For iOS you additionally need to set the CMake cache variable `MOLTEN_VK_SDK` to the root of MoltenVK inside the Vulkan SDK.
```

Comment:
Because when the CMake GUI is run from macOS Launchpad it will not see environment variables, VULKAN_SDK needs to be a CMake variable initialized from $ENV{VULKAN_SDK} if it is visible. The instructions on these lines need to reflect this. e.g set the CMake cache variable VULKAN_SDK if it is not already set.